### ISSUE TITLE
missing jquery when project updated to ember 3.4

### ISSUE THREAD

**Issue by csantero** [san650/ember-cli-page-object#423]
I've tried updating an addon of mine which uses ember-cli-page-object from ember-source 3.1 to 3.4, and have encountered a problem regarding jquery. My addon imports a jquery plugin that relies on a global `jQuery` object being present. Post-update, even with `@ember/jquery enabled`, ember-cli-page-object is including the `ecpo-jquery` shim, which _deletes_ the global `jQuery` that the jquery plugin is relying on. Before updating, this global object was provided by ember, and e-c-p-o was including the `project-jquery` vendor shim, which leaves the global alone.

Is this a problem anyone else has run into? Maybe this could be solved by checking in index.js for whether the `jquery-integration` optional feature is enabled, and if so, using `project-jquery` instead of `ecpo-jquery`?

**Comment by san650** [comment]
@csantero is the addon you're updating public? If so, can you post a link to the branch that reproduces this issue?

**Comment by san650** [comment]
> I also haven't seen the error again since trying 1.15.0-beta.3 (I had been using 1.14.1)

This is good to hear, we're planning on releasing v1.15.0 soon.

> I'd still consider it to be undesirable behavior for e-c-page-object to be deleting the global jQuery object provided by @ember/jquery.

Completely agreed. Do you know how can we fix this issue? Could you try fixing this issue in ecpo? I could give it a try at fixing this but not sure how to reproduce the issue. I would appreciate any help!

**Comment by jrjohnson** [comment]
I was seeing this as well and created a reproduction at 

https://github.com/jrjohnson/reproduction-page-object-423
It is just 
```
ember new app --yarn
ember install ember-cli-page-object
ember install ember-froala-editor
```
`yarn test` will cause the error. It is broken all the way through 1.15 beta 2, but it is fixed in beta 3.

**Comment by ro0gr** [comment]
recently released v1.15 should have a fix for that. Please try it out, and re-open if the issue persists.
